app/containers/CustomIcon/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { createIconSetFromIcoMoon } from 'react-native-vector-icons';
+import { TextProps } from 'react-native';
+
+import { mappedIcons } from './mappedIcons';
+
+const icoMoonConfig = require('./selection.json');
+
+export const IconSet = createIconSetFromIcoMoon(icoMoonConfig, 'custom', 'custom.ttf');
+
+export type TIconsName = keyof typeof mappedIcons;
+
+interface ICustomIcon extends TextProps {
+	name: TIconsName;
+	size: number;
+	color?: string;
```

Comment:
It should be required.
With this change, TS is going to complain a little bit with the styles.

Pay attention to:

```tsx
// before
<CustomIcon name='close' style={[styles.close, { color: themes[theme].titleText }]} size={20} />
...
// after: not only add the `color` prop, but remove the array from `style` as well
<CustomIcon name='close' style={styles.close} color={themes[theme].titleText} size={20} />
```

Reply:
due this change, I had to update the snapshots removing the colors from the style, can you take a look ?